### ISSUE TITLE
BatchNormalization layer fails with torch backend on GPU with `scale=False, center=True`

### ISSUE THREAD

**Issue by haifeng-jin** [keras-team/keras#19073]
Here is the colab for reproducing it.
[link](https://colab.research.google.com/github/haifeng-jin/Colabs/blob/main/Keras_torch_BN.ipynb)

**Comment by haifeng-jin** [comment]
The bug is fixed by https://github.com/keras-team/keras/commit/9815ac1c81f0f24cf3ca24638e4ab11c5fc95bd1.
However, the torch BN is converging slower than other backends. Needs further investigation.
